### DISCUSSION TITLE
How to obtain the original URL of the pre training model and manually download it

### DISCUSSION THREAD

**Discussion by au-top** [coqui-ai/TTS#2439] [General Q&A] [answered]
Sorry, I'm stuck at the beginning for some network reason
```
tts --text "hello world"
```
tts  unable to download due to network reasons
I need to manually download  model . tts I can't automatically download the model for me
I need to manually obtain the source address of the pre training model and manually download it, but after checking the documentation, I cannot find relevant information. How can I obtain the model resource address and allow me to manually download and install it?

**Comment by thorstenMueller** [comment, marked as answer]
You can take the download url from .models.json file.
Example: https://github.com/coqui-ai/TTS/blob/14c80dd1fd665ea348ce83f2b34392485ac5dfa6/TTS/.models.json#L7
